include/onnxruntime/core/session/onnxruntime_cxx_api.h

```diff
@@ -370,7 +370,7 @@ struct Value : Base<OrtValue> {
   const T* GetTensorData() const;
 
   template <typename T>
-  T At(const std::initializer_list<size_t>& location);
+  T At(const std::initializer_list<int64_t>& location);
```

Comment:
Shouldn't this return a T& to expose the C API fully?

Reply:
I think you are right.

---
In reply to: [488952286](https://github.com/microsoft/onnxruntime/pull/5180#discussion_r488952286) [](ancestors = 488952286)